docs/usage/cli.md

```diff
@@ -20,6 +20,14 @@ available from the command line.
 $ npm install --global babel-cli
 ```
 
+Out of the box, Babel (since v6) doesn't perform any transformation, it only validates the syntax. You may want to install [plugins](/docs/plugins) or more likely whole [presets](/docs/plugins/#presets) before going further. For the sake of the example, we'll use the [es2015](/docs/plugins/preset-es2015/) preset.
+
+```sh
+$ npm install babel-preset-es2015
```

Comment:
Serious users already know about npm scripts. Adding these parameters add noise and clobber the flow of the article.

The article is a tutorial for using Babel from the command line. For dabbling. To get a feel of how the thing works. It's pretty good as it is.

All examples show interactive commands and hint at output, and it's great as is. 

If babel really shouldn't be used from the CLI at all, you may want to rename `babel-cli` `babel-npm` and refuse to run outside of `npm` scripts.